src/analysis/normed_space/ray.lean

```diff
@@ -80,12 +76,8 @@ lemma same_ray_iff_norm_smul_eq : same_ray ℝ x y ↔ ∥x∥ • y = ∥y∥ 
 vectors `∥x∥⁻¹ • x` and `∥y∥⁻¹ • y` are equal. -/
 lemma same_ray_iff_inv_norm_smul_eq_of_ne (hx : x ≠ 0) (hy : y ≠ 0) :
   same_ray ℝ x y ↔ ∥x∥⁻¹ • x = ∥y∥⁻¹ • y :=
-begin
-  have : ∥x∥⁻¹ * ∥y∥⁻¹ ≠ 0, by simp *,
-  rw [same_ray_iff_norm_smul_eq, ← smul_right_inj this]; try { apply_instance },
-  rw [smul_comm, mul_smul, mul_smul, smul_inv_smul₀, inv_smul_smul₀, eq_comm],
-  exacts [norm_ne_zero_iff.2 hy, norm_ne_zero_iff.2 hx]
-end
+by rw [inv_smul_eq_iff₀, smul_comm, eq_comm, inv_smul_eq_iff₀, same_ray_iff_norm_smul_eq];
+    rwa norm_ne_zero_iff
```

Comment:
Can you combine these rws? If not, I think we tend to avoid using `a; b` in places where `{ a, b }` would work

Reply:
The second `rwa` rewrites in two goals generated by rewriting on `inv_smul_eq_iff₀`.

Reply:
@eric-wieser :arrow_up: